bigquery/google/cloud/bigquery/_http.py

```diff
@@ -22,11 +22,9 @@
 class Connection(_http.JSONConnection):
     """A connection to Google BigQuery via the JSON REST API.
 
-    :type client: :class:`~google.cloud.bigquery.client.Client`
-    :param client: The client that owns the current connection.
+        client (:class:`~google.cloud.bigquery.client.Client`): The client that owns the current connection.
```

Comment:
No `:class:` for the types in Google-style.

```suggestion
        client (google.cloud.bigquery.client.Client): The client that owns the current connection.
```